<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>431</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="1">
     <widget class="QPushButton" name="excel_savebtn">
      <property name="font">
       <font>
        <family>나눔스퀘어라운드 Regular</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>저 장</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="showfilename">
      <property name="font">
       <font>
        <family>나눔스퀘어라운드 Regular</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="setsheet">
      <property name="font">
       <font>
        <family>나눔스퀘어라운드 Regular</family>
       </font>
      </property>
      <item>
       <property name="text">
        <string>sheet1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sheet2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>나눔스퀘어라운드 Bold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;엑셀 파일 불러오기&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>나눔스퀘어라운드 ExtraBold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Excel File 업로드</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>나눔스퀘어라운드 Bold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;DB 이름 설정하기&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="excel_homebtn">
      <property name="font">
       <font>
        <family>나눔스퀘어라운드 Bold</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="excel_actbtn">
        <property name="font">
         <font>
          <family>나눔스퀘어라운드 Regular</family>
         </font>
        </property>
        <property name="text">
         <string>실 행</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="excel_closebtn">
        <property name="font">
         <font>
          <family>나눔스퀘어라운드 Regular</family>
         </font>
        </property>
        <property name="text">
         <string>취 소</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="showtext">
      <property name="font">
       <font>
        <family>나눔스퀘어라운드 Regular</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="excel_importbtn">
      <property name="font">
       <font>
        <family>나눔스퀘어라운드 Regular</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>불 러 오 기</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="setname">
      <property name="font">
       <font>
        <family>나눔스퀘어라운드 Regular</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="selectbtn">
      <property name="font">
       <font>
        <family>나눔스퀘어라운드 Regular</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>선 택</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="showdbname">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
